Tools/WinMLRunner/src/CommandLineArgs.cpp

```diff
@@ -69,7 +70,7 @@ void CheckAPICall(int return_value)
 
 CommandLineArgs::CommandLineArgs(const std::vector<std::wstring>& args)
 {
-    for (UINT i = 0; i < args.size(); i++)
```

Comment:
Since we don't expect i<0 to be used. Would it be okay to change this type to uint32_t?

Reply:
args.size() returns a size_t (which is signed).  Kept the variable the same to prevent the warning.  I could also call it size_t to explicitly match the output type of args.size(), or cast it if you would prefer that.

Reply:
Seems like according to this: https://stackoverflow.com/questions/409348/iteration-over-stdvector-unsigned-vs-signed-index-variable

std::vector<std::wstring>::size_type is the appropriate type for the index variable? thoughts? The function signature of CheckNextArgument may need to be changed to relect this.

Reply:
I think that making that change is fine.  Will proceed.